specifyweb/frontend/js_src/lib/components/FormCells/FormTable.tsx

```diff
@@ -137,9 +138,8 @@ export function FormTable<SCHEMA extends AnySchema>({
     const lastRow: HTMLElement | null = rowsRef.current.querySelector(
       `:scope > :nth-child(${resourceIndex + 1}) > [tabindex="-1"]`
     );
-    const firstFocusableElement = rowsRef.current.querySelector<HTMLElement>(
-      'a, input:not([type="hidden"]), select, textarea, [tabindex]:not([tabindex="-1"])'
-    )!;
+    lastRowRef.current = lastRow;
+    const firstFocusableElement = findFirstFocusableElement(lastRowRef);
```

Comment:
(optional) lastRowRef shouldn't be needed
you could modify findFirstFocusableElement to accept `HTMLElement | null`
or pass `{current: lastRow}` as an argument

Reply:
The logic looks good. Does this code still not work? Do you know why? if not, I can take a look at it